src/index.ts

```diff
@@ -0,0 +1,291 @@
+  // one method can be used to update any errGenerator data
+  // attribute by passing key/value pairs to the following function
+  updateData(key, value) {
+    // @ts-ignore
+    errGenerator.data[key] = value;
+  },
+  defaultReport(level, message, response) {
+    errGenerator.report(level, message, {
+      status: response.status,
+      error: response.data,
+      config: {
+        method: response.config.method,
+        url: response.config.url,
+        data: isObjectLike(response.config.data)
+          ? JSON.stringify(response.config.data)
+          : response.config.data,
+      },
+      requestDuration: response.config.duration,
+    });
+  },
+  // Method to send generated error report to error logging service
+  report(level, message, extraData) {
+    const loggedData = extraData || {};
+    if (typeof message === 'string' && message.length > 0 && message.indexOf('Script error.') > -1) {
+      return;
+    }
+
+    loggedData.location = window.location.href;
+    loggedData.clientData = genClientBlock(window);
+    loggedData.userData = userBlock(errGenerator.data.self);
+    loggedData.additional = additionalBlock(
+      errGenerator.data.version,
+      errGenerator.data.commit,
+      errGenerator.data.timeOffset,
+      errGenerator.data.startTime,
+    );
+
+    if (
+      errGenerator.data.loggly.key
+      && isAllowedErrorLevel(
+        level,
+        errGenerator.data.loggly.loggingLevel,
+        errGenerator.data.errorLevelOrder,
+      )
+    ) {
+      reportToLoggly(
+        level,
+        JSON.stringify(message, Object.getOwnPropertyNames(message)),
+        loggedData,
+      );
+    }
+  },
+};
+
+// Check if current error should be reported
+function isAllowedErrorLevel(errorLevel: ErrorLevel, loggingLevel: ErrorLevel, possibleLevels: ErrorLevel[]) {
+  if (possibleLevels.indexOf(loggingLevel) === -1) {
+    return true;
+  }
+  return (
+    possibleLevels.indexOf(loggingLevel) >= possibleLevels.indexOf(errorLevel)
+  );
+}
+// A function to register loggly
+function registerLoggly(loggly: ILoggly) {
+  // @ts-ignore
+  errGenerator.data._LTracker = window._LTracker || [];
+  errGenerator.data._LTracker.push({
+    logglyKey: loggly.key,
+    sendConsoleErrors: true,
+    tag: loggly.tag,
+  });
+
+  // This is extension of tracker prototype track function. In previous versions
+  // of loggly-jslogger this function had possibility to add fallback.
+  // This is possible to see in admin where older version is being used.
+  // This fallback helps us to guarantee to send error if for some reason
+  // first time it didn't reach the service.
+  // ****
+  // Currently used version:
+  // https://cloudfront.loggly.com/js/loggly.tracker-2.2.2.js
+  // ****
+  // Code was taken from latest version (Content-type):
+  // https://cloudfront.loggly.com/js/loggly.tracker-latest.js
+  // @ts-ignore
+  errGenerator.data._LTracker.track = function (this: any, data: any) {
```

Comment:
Could you check with @MariusV-ADV if this hack is still necessary, maybe there is a newer version of loggly that has this implementation back?
![image](https://user-images.githubusercontent.com/20832646/188576759-4b3da786-e097-48ab-a70c-a9eb4d88846e.png)

Reply:
I've checked newest version of loggly (2.2.4), they're still using this hack in their min.js.